[feat] Add type local turn on resource center by default (#13303)
apache/dolphinscheduler: docs/docs/en/guide/resource/configuration.md

### Configure `common.properties`

> Dolphinscheduler Resource Center uses the local system is enabled by default, and does not require any additional configuration.

Reviewer: ```suggestion
DolphinScheduler Resource Center uses local file system by default, and does not require any additional configuration.
```
